src/Processors/QueryPlan/Optimizations/QueryPlanOptimizationSettings.h

```diff
@@ -30,6 +30,9 @@ struct QueryPlanOptimizationSettings
     /// If aggregation-in-order optimisation is enabled
     bool aggregation_in_order = false;
 
+    /// true if removing redundant DISTINCT clauses is enabled
```

Comment:
(Really minor: Could use some consistency with the previous comments: `If removal of DISTINCT clauses is enabled.`).